.env.sample

```diff
@@ -53,4 +53,8 @@ KAFKA_BINANCE_DATA_TOPIC=binance_data_topic
 
 COVALENT_API_KEYS='key1,key2,key3'
 
-MOCHI_PROFILE_SERVER_HOST="https://api.mochi-profile.console.so"
\ No newline at end of file
+MOCHI_PROFILE_SERVER_HOST="https://api.mochi-profile.console.so"
+
+PKPASS_MOCHI_PASS_TEAM_IDENTIFIER=""
+PKPASS_MOCHI_PASS_TYPE_IDENTIFIER=""
+PKPASS_MOCHI_ORGANIZATION_NAME=""
```

Comment:
we can fill in random text for these?

Reply:
I don;t think so, this specific with the cert information, now we not have stg env value